smart_card_connector_app/src/chrome-api-jstocxxtest.js

```diff
@@ -46,7 +46,8 @@ let pcscReadinessTracker;
 let mockChromeApi;
 /** @type {!goog.testing.MockControl|undefined} */
 let mockControl;
-
+/** @type {ChromeApiProvider?} */
+let chromeApiProvider;
```

Comment:
nit: Restore the blank line before the next function.

Reply:
thanks :)